### PR TITLE
[Merged by Bors] - feat(RingTheory/Kaehler): The exact sequence `B ⊗[A] Ω[A⁄R] → Ω[B⁄R] → Ω[B⁄A] → 0`.

### PR DESCRIPTION
**Pull request by erdOne**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by riccardobrasca — COMMENTED** [review]
Do we have enough technology to use [CategoryTheory.ShortComplex.ShortExact](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/Homology/ShortComplex/ShortExact.html#CategoryTheory.ShortComplex.ShortExact) or something like that? I mean, literally saying the sequence is exact.

**Comment by joelriou** [comment]
> Do we have enough technology to use [CategoryTheory.ShortComplex.ShortExact](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/Homology/ShortComplex/ShortExact.html#CategoryTheory.ShortComplex.ShortExact) or something like that? I mean, literally saying the sequence is exact.

In general, we cannot use category theory here because of heterogeneous universes. Eventually, it may be useful to state exactness (by saying that a certain cokernel cofork is colimit) when all the modules are in the same universe, but I do not feel it has to be done in this PR.

Exactness could also be formulated also using the basic API in `Mathlib.Algebra.Exact` which allow different universes.

File: Mathlib/RingTheory/Kaehler.lean [hunk 1]
```diff
@@ -650,13 +660,37 @@ theorem KaehlerDifferential.map_compDer :
...
 
+@[simp]
 theorem KaehlerDifferential.map_D (x : A) :
     KaehlerDifferential.map R S A B (KaehlerDifferential.D R A x) =
       KaehlerDifferential.D S B (algebraMap A B x) :=
   Derivation.congr_fun (KaehlerDifferential.map_compDer R S A B) x
 set_option linter.uppercaseLean3 false in
 #align kaehler_differential.map_D KaehlerDifferential.map_D
 
+unsuppress_compilation in
```
[the remark this answers is not in the document]
Comment (erdOne) [reply Mathlib/RingTheory/Kaehler.lean:671]: It is to undo the `suppress_compilation` at the top of the file (which is for performance reasons)
As for why notations need compilation, I have no idea.

**Comment by riccardobrasca** [comment]
Can you add a short docstring as I wrote above? Anyway thanks a lot!

bors d+

**Comment by erdOne** [comment]
Sorry it took so long to fix. I've incorporated some suggestions above and a few more and maybe @riccardobrasca can give it a final look before I send it to bors?

**Comment by riccardobrasca** [comment]
> Sorry it took so long to fix. I've incorporated some suggestions above and a few more and maybe @riccardobrasca can give it a final look before I send it to bors?

I am boarding on a long flight, I can have a look tomorrow.

**Pull request closed without merging**